driver/src/main/scala/api/commands/AtlasSearchAggregation.scala

```diff
@@ -1242,37 +1283,37 @@ private[commands] trait AtlasSearchAggregation[P <: SerializationPack] {
 
         override def toString = tpe
       }
+    }
 
-      sealed trait Writer[T] {
-        /** Returns the serialized representation for the input value `v` */
-        def write(v: T): pack.Value
-      }
+    sealed trait Writer[T] {
```

Comment:
Please keep the encapsulation level

Reply:
you mean to keep Writer in api package?

Reply:
It was inside `Range`, just keep it there